.github/workflows/dev_actions.yml

```
@@ -4,59 +4,123 @@
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 3.1.x
+
+      - name: Setup .NET Core 8.0
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 8.0.x
+
+      - name: Setup MSBuild Path
+        uses: microsoft/setup-msbuild@v1.1
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)